trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core.tests/src/org/eclipse/tracecompass/incubator/trace/server/jersey/rest/core/tests/stubs/TmfConfigurationSourceTypeStub.java

```diff
@@ -66,33 +71,32 @@ public TmfConfigurationSourceTypeStub(@JsonProperty("id") String id,
             builder.setConfigParamDescriptors(parameterDescriptors.stream().map(stub -> stub.getConfig()).collect(Collectors.toList()));
         }
         fConfig = builder.build();
+        fSchema = schema;
```

Comment:
Should it be added to the builder and to TmfConfigurationSourceType class?

Reply:
This would that TmfConfigurationSourceType would require the jackson JSON parser because `fSchema` is of type `JsonNode. Besides the `TmfConfigurationSourceTypeStub` a client-side implementation for testing the trace server as part of the unit tests.

Reply:
Ok, if I understand correctly, this stub class is on the client side of the test, not the trace-server side, and it uses TmfConfigurationSourceType as simplification to deserialize the trace-server response?

Reply:
yes, it's on the client side of the test.